services/getUserIdsFromRoleId.js

```diff
@@ -0,0 +1,21 @@
+const firestore = require("../utils/firestore");
+const memberRoleModel = firestore.collection("member-group-roles");
+const usersModel = firestore.collection("users");
+
+export const getUserIdsFromRoleId = async (roleId) => {
+  let discordIds = [];
+  const querySnapshot = await memberRoleModel.where("roleid", "==", roleId).get();
```

Comment:
```suggestion
const querySnapshot = await memberRoleModel.where("roleid", "==", roleId).get();
if (querySnapshot.empty) {
    return [];
  }
```
we can return early if the snapshot is empty